docs/pages/components/tables.vue

```diff
@@ -559,13 +575,69 @@ slot-scope is not required for the `loading` and `empty` slots
 </KtTable>
 ```
 
+### Provider/Consumer and Mixin
+
+Sometimes you may need to access the table's store and control it from outside,
+while `ref` may work if you're modifications are in the same component, you controling ui may be elsewhere
+For that purpose we introduce `KtTableProvider` `KtTableConsumer` and optionally but discourged we expose `KtTableColumnsStateMixin`
+
+<KtTableProvider>
+	<div>
+		<KtTableConsumer #default="{ columns, hideColumn }">
```

Comment:
I’m assuming that’s a typo. Also, next time please don’t use a reserved keyword for an API.

```suggestion
		<KtTableConsumer default="{ columns, hideColumn }">
```